http_request/http_request.py

```diff
@@ -105,6 +101,33 @@ def addKeyToObject(self, key="", msg="", data={}):
     def formatJson(self, data):
         return str(data).strip("'<>() ").replace("'", '"')
 
+    def redaction(self, msg):
+        REDACTED = '[REDACTED]'
+
+        msg = str(msg)
```

Comment:
might as well put this inside the `try:`, as it's possible that something weird gets passed to the `redaction` function and string casting breaks